src/ProgressOnderwijsUtils/Extensions/StringExtensions.cs

```diff
@@ -77,10 +75,11 @@ public static string Replace(this string s, [NotNull] IEnumerable<KeyValuePair<s
         /// 'S-GRAVENHAGE -> 's-Gravenhage
         /// </summary>
         [Pure]
-        public static string ToCamelCase(this string s)
+        [return: NotNullIfNotNull("s")]
+        public static string ToCamelCase(this string? s)
         {
             if (s.IsNullOrWhiteSpace()) {
-                return s;
+                return s!;

```

Comment:
waarom is dit nodig?

Reply:
Geen idee; de compiler niet slim genoeg is.

Reply:
Misschien is het logischer het return type nullable te maken.

Reply:
Of zowel invoer als uitvoer non-nullable, en dan ga je maar `?.` gebruiken, of, worst-case, `.NullIfWhiteSpace()?.ToCamelCase()` als het oude gedrag echt belangrijk is.

Reply:
(ooit dan eh: lijkt mee en super-goed idee eerst even het bestaande gedrag te annoteren zoals je nu doet, met alle onhebbelijkheden er gewoon nog in, om fixes duiderlijk toepaspaar te maken).